_includes/content/compose-matrix.md

```diff
@@ -2,6 +2,7 @@ This table shows which Compose file versions support specific Docker releases.
 
 | **Compose file format** | **Docker Engine release** |
 |  -------------------    |    ------------------     |
+|  Compose specification  |       19.03.0+            |
```

Comment:
Docker 20.10 will ship with with compose-schema 3.9 (https://github.com/docker/cli/tree/v20.10.0-beta1/cli/compose/schema/data). Not sure how we'll address that, and it the compose-specification has all options that were added in that, so we probably need another matrix somewhere to list what's supported for `docker stack` commands.

I'm wondering if we need to come with a different name for this ("unified schema" ?) and/or if there will be releases of the compose-specification, otherwise it will be impossible to know what's part of the schema and what not.